build.gradle

```diff
@@ -65,12 +70,14 @@ jar {
     }
 }
 
-processResources {
-    into "lib/jni/", {
-        from ("lib/jni/libIntelDeflater.so")
-    }
+task extractIntelGKL(type: Copy) {
```

Comment:
i think you can remove this task. Just make sure to check if the so/dylib is still there in the jar